cmd/minikube/cmd/start.go

```diff
@@ -1185,6 +1191,31 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 	}
 }
 
+// validateBaseImage checks that --base-image is not passed if the drive being in use is KIC (docker/podman)
+// if so, the function exits the process
+func validateBaseImage(imageFlag *pflag.Flag, driver string) {
```

Comment:
to avoid confusion with driver package consider using a shorter variable name for driver, such as `drv`

Reply:
good catch! fixed.